docs/sources/reference/components/pyroscope/pyroscope.ebpf.md

```diff
@@ -17,6 +17,12 @@ To use the  `pyroscope.ebpf` component you must run {{< param "PRODUCT_NAME" >}}
 
 You can specify multiple `pyroscope.ebpf` components by giving them different labels, however it's not recommended as it can lead to additional memory and CPU usage.
 
+## Supported languages
+
+This eBPF profiler only collects CPU profiles. Generally natively compiled languages like C/C++, Go, Rust are supported (also see [Troubleshooting]({{< relref "#troubleshooting" >}}) for additional requirements).
+
+The only high-level language supported is Python (as long as `python_enabled=true`). Other high-level languages like Java, Ruby, PHP, JavaScript, etc. will require some additional work in order to correctly show stack traces of methods in these languages. Currently their CPU usage will be displayed belonging to the runtime's methods instead.
```

Comment:
```suggestion
Python is the only supported high-level language, as long as `python_enabled=true`. Other high-level languages like Java, Ruby, PHP, and JavaScript require additional work to show stack traces of methods in these languages correctly. Currently, their CPU usage will be displayed belonging to the runtime's methods instead.
```

Where is the CPU usage displayed? Who is displaying it? Alloy? Grafana?

Reply:
I guess Grafana is displaying it in the flamegraphs, maybe we us reported, as that is what Alloy is doing?

```suggestion
Python is the only supported high-level language, as long as `python_enabled=true`. Other high-level languages like Java, Ruby, PHP, and JavaScript require additional work to show stack traces of methods in these languages correctly. Currently, their CPU usage will be reported belonging to the runtime's methods instead.
```